### PR TITLE
[Makefile] Produce reproducible docker-compose

### PR DESCRIPTION
**Pull request by hallelujah**
- Adds volume to keep assets and node_modules
- Remove RAILS_ENV=development to allow all db creation
- Remove unused targets

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,104 +27,71 @@ RUBY_ENV += RUBY_GC_OLDMALLOC_LIMIT_GROWTH_FACTOR=1.2
...
-tmp-export: clean-tmp
-	-@ $(foreach dir,$(TMP),docker cp $(IMAGE):/opt/system/$(dir) $(dir) 2>/dev/null;)
-
-clean-tmp: ## Removes temporary files
-	-@ $(foreach dir,$(TMP),rm -rf $(dir);)
+assets: ## Create assets volumes
+assets: yarn
+assets: CMD = rake assets:precompile
+yarn: ## Install JS dependencies
+yarn: CMD = yarn install
```
Comment (didierofrivia) [inline Makefile]: I know Yarn is the standard of Rails 5 (I think from Rails 5.1), but this won't work in our current setup. We need to migrate back to Yarn our Webpack entire suite.

Comment (hallelujah) [reply Makefile]: Why won't it work?

**Comment by kevprice83** [comment]
@hallelujah this worked for me. I only had an issue with installing all the yarn packages due to network being slow...not sure that was really the case though so I set `--network-timeout=100000` and it was fixed for me, maybe many packages and some quite large? If nobody else has this issue then this can be ignored.

**Comment by hallelujah** [comment]
@kevprice83 I did not have any issue but only on migration where mysql was not completely up so it fails
Re-running dev-start again worked

**Pull request merged**